### PR TITLE
add stop logic: docker-compose down

### PR DESCRIPTION
**Pull request by zchcai**
Fix https://github.com/google/fuzzbench/pull/588#discussion_r461247543

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -32,6 +32,18 @@ run-end-to-end-test:
 	docker-compose run run-tests; STATUS=$$?; \
 	docker-compose down; exit $$STATUS
 
+# Stop and clean running environment.
+
+stop-experiment: export COMPOSE_PROJECT_NAME := fuzzbench
```
Comment (jonathanmetzman) [inline Makefile:37]: Can you make this one target please?

File: Makefile [hunk 2]
```diff
@@ -32,6 +32,18 @@ run-end-to-end-test:
...
+stop-experiment: export COMPOSE_FILE := compose/fuzzbench.yaml
+stop-experiment:
+	docker-compose down
+
+stop-end-to-end-test: export COMPOSE_PROJECT_NAME := e2e-test
```
Comment (jonathanmetzman) [inline Makefile:42]: Same comment.

File: Makefile (hunk 1 above)
Comment (zchcai) [reply Makefile:37]: I guess different COMPOSE_PROJECT_NAME needs different targets?

Comment (zchcai) [reply Makefile]: Done.

File: Makefile [hunk 3]
```diff
@@ -32,6 +32,14 @@ run-end-to-end-test:
 	docker-compose run run-tests; STATUS=$$?; \
 	docker-compose down; exit $$STATUS
 
+# Stop and clean running environment.
+
+stop-experiment:
```
Comment (lszekeres) [inline Makefile:37]: You can 1) re-use the env var settings above, and 2) mention that this is only necessary to run if the experiment was interrupted and the the down command above didn't run. I.e.:

```
run-experiment stop-experiment: export COMPOSE_PROJECT_NAME := fuzzbench
run-experiment stop-experiment: export COMPOSE_FILE := compose/fuzzbench.yaml
run-experiment:
	docker-compose up --build --scale worker=2 --detach
	docker-compose logs --follow run-experiment
	docker-compose down

# Running this is only necessary if `run-experiment` was interrupted and
# containers were not cleaned up.
stop-experiment:
	docker-compose down
```

Comment (zchcai) [reply Makefile:37]: What about `stop end-to-end-test`? Shall we omit that?

Comment (lszekeres) [reply Makefile:37]: You can add  a `stop-end-to-end-test` rule below `run-end-to-end-test` in a similar fashion, if you think that's useful to have. But please add the same comment saying that in general explicit stopping is not necessary.

Comment (zchcai) [reply Makefile:37]: OK. @lszekeres  As suggested in https://github.com/google/fuzzbench/pull/600#discussion_r461709175
I think @jonathanmetzman  suggest only one target. :)

Comment (lszekeres) [reply Makefile:37]: Not sure what that comment was referring to, but "run-experiment" and "run-end-to-end-test" are two separate setups. If we'd like to have a manual cleanup target for both of them, please let's add a corresponding "stop-experiment" and "stop-end-to-end-test" target, as described above.

Comment (jonathanmetzman) [reply Makefile:37]: My comment was sort of a misunderstanding, I thought there were a bunch of lines listing dependencies for the stop-experiment target that could be merged into one line listing dependencies.

Comment (zchcai) [reply Makefile]: I see. Done.

**Pull request merged**